docs/api-reference/api-reference-guide.rst

```diff
@@ -268,7 +271,6 @@ layout_t
 ^^^^^^^^
 
 .. doxygenenum:: rocwmma::layout_t
-   :members:
```

Comment:
doxygenenum does not have members property. The output is correct after removing it.

![image](https://github.com/user-attachments/assets/fc40b146-95e6-4d1f-b31c-15d3e04be898)